cloudferrylib/os/network/neutron.py

```diff
@@ -279,7 +279,6 @@ def convert_subnets(snet, cloud):
         }
 
         res_hash = network_res.get_resource_hash(result,
-                                                 'name',
```

Comment:
This doesn't seem right to me right now. Why are we not comparing name of a network?..